nearcore/src/migrations.rs

```diff
@@ -86,6 +91,11 @@ impl<'a> near_store::StoreMigrator for Migrator<'a> {
             37 => near_store::migrations::migrate_37_to_38(store),
             38 => near_store::migrations::migrate_38_to_39(store),
             39 => near_store::migrations::migrate_39_to_40(store),
+            40 => near_store::migrations::migrate_40_to_41(
+                store,
+                kind.unwrap_or_else(|| panic!("DB kind is not set for DB at version: {}", version)),
```

Comment:
nit: `.expect(format!(...))` would be a bit more typical

Reply:
I tried using it before, but compiling `expect` in that form fails due to calling a function in expect, so if I need formatting with arguments, I use `unwrap_or_else(panic!(...))`.